assets/stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,56 @@
+$base-font-face: "acumin-pro", helvetica, arial, sans-serif;
+$condensed-font-face: "acumin-pro-extra-condensed", sans-serif;
+
+$base-font-size: 14px;
+
+@mixin font-style($face: $base-font-face, $size: $base-font-size, $weight: normal) {
```

Comment:
Eh, the `crds` convention was something I thought of today. I'm still trying to figure out the best approach for naming Crossroads-specific styles - before I was just overriding Bootstrap-named classes. I'm still not sure which would be the best approach tbh.